### PR TITLE
auto-dark-light@gihaume:  replaced `DateTime` methods incompatible with GLib < 2.80

### PR DESCRIPTION
**Pull request by guillaume-mueller**

### CODE REVIEW

**Comment by guillaume-mueller** [comment]
Sorry I'm not sure why my previous commit is included.

Should I delete the branch and do a new one for each pull request ?

**Comment by rcalixte** [comment]
> Should I delete the branch and do a new one for each pull request ?

No need. We can fix it here or you can just rebase.

File: auto-dark-light@gihaume/files/auto-dark-light@gihaume/lib/twilights_calculator.js [hunk 1]
```diff
@@ -17,14 +17,15 @@ const Twilights_calculator = {
...
+        const now   = DateTime.new_now_local().to_unix() * 1000, // [ms]
               dates = SunCalc.getTimes(now, latitude, longitude);
 
         if (isNaN(dates.sunrise) || isNaN(dates.sunset))
             throw Error(_("unable to calculate twilight times, check coordinates format or range"));
 
         const [sunrise, sunset] = [dates.sunrise, dates.sunset].map(date => {
-            const date_glib = DateTime.new_from_unix_local_usec(date.getTime() * 1000);
+            const unix_time = date.getTime() / 1000, // [s]
+                  date_glib = DateTime.new_from_unix_local(unix_time);
```
Comment (rcalixte) [inline auto-dark-light@gihaume/files/auto-dark-light@gihaume/lib/twilights_calculator.js:28]: This syntax doesn't seem correct?

Comment (guillaume-mueller) [reply auto-dark-light@gihaume/files/auto-dark-light@gihaume/lib/twilights_calculator.js:28]: What doesn't seem correct ?

**Comment by guillaume-mueller** [comment]
> > Should I delete the branch and do a new one for each pull request ?
> 
> No need. We can fix it here or you can just rebase.

Ah I think it is because I haven't merged my branch to the `master` one before the commit.

**Comment by rcalixte** [comment]
> Ah I think it is because I haven't merged my branch to the `master` one before the commit.

Once you pull/sync your forked master and rebase this branch, you can force push and the extra commit should go away.

File: auto-dark-light@gihaume/files/auto-dark-light@gihaume/lib/twilights_calculator.js (hunk 1 above)
Comment (rcalixte) [reply auto-dark-light@gihaume/files/auto-dark-light@gihaume/lib/twilights_calculator.js:28]: There isn't a semicolon on the line 27 and line 28 doesn't have a declaration.

Comment (rcalixte) [reply auto-dark-light@gihaume/files/auto-dark-light@gihaume/lib/twilights_calculator.js:28]: Ah, sorry, I see now. It should work. The formatting is just unusual for me to read. 😅

**Pull request merged**